pages/content/posts/complete-interop-features/index.md

```diff
@@ -0,0 +1,66 @@
+
+But, since the early 2010s, and even a bit earlier, standard bodies and browser makers rapidly specified and implemented new APIs. The focus was to document these new features: they were the focus of web developers, and there needed to be more good documentation about them! It was an easier move to sell internally: new technologies lead to more engagement; getting resources to document them was more straightforward than the old HTML table features, for example. The HTML5, CSS3, and ES6 initiatives allowed MDN to revamp these areas of documentation as there was an interest in positioning the web as a recognized interoperable platform.
+
+![The three logos of HTML5, CSS3, and JS](logos.png)
+
+So, the combination of the scarcity of resources and the high-speed development of the web platform led to the original gaps staying open. Some mitigating actions were taken, like listing every method and property for each interface, but there was still a large gap (more than 1000 missing pages).
+
+Today's gaps mostly fall into three categories:
+
+- HTML DOM and the SVG DOM features added to the web platform when MDN was still in its infancy;
+- non-interoperable features: APIs supported by only one or two browser engines and missed by the browser makers' writers;
+- single properties and methods forgotten when their API was initially documented or added later (but they slipped below the radar of the different writing processes).
+
+## Opportunities created by the Interop initiative
+
+As a collective of writers dedicated to helping document the open web platform, [Open Web Docs (OWD)](https://openwebdocs.org/) can help in _closing some of these gaps_. Unlike writers paid by browser makers needing to justify their work, our backers, individuals or companies, gave us the more generic mission to improve the open web documentation. We focus on much-needed activities that do not necessarily bring the most engagement at first, though improving in the middle term situation significantly. We still need to focus on the most valuable activities, but anybody's internal projects do not dictate our priorities.
+
+For several years now, browser makers have sat together at the end of each year to discuss the pain points of web developers and define a set of features that needs more love to increase their functionality or to fix bugs and incoherence so that web devs can use them reliably. They define areas to focus on as part of the [Interop](https://web.dev/interop-2023/) initiative.
+
+OWD, and its partners, think these projects are good opportunities to revisit the relevant area of the MDN Web Docs documentation: the idea is to have the docs in good shape for each set of features made interoperable over the year. Most of the time, this leads to minor changes to the docs: MDN writers tried to describe the interoperable behaviour, only mentioning browsers when their implementations diverged.
```

Comment:
```suggestion
OWD and its partners think these projects are good opportunities to revisit the relevant area of the MDN Web Docs documentation. The idea is to have the docs in good shape for each set of features made interoperable over the year. Most of the time, this leads to minor changes to the docs. MDN writers tried to describe the interoperable behavior, such as only mentioning browsers when their implementations diverged.
```

Reply:
I prefer the English spelling. (We are not on MDN where we use the American spelling)
```suggestion
OWD, and its partners, think these projects are good opportunities to revisit the relevant area of the MDN Web Docs documentation. The idea is to have the docs in good shape for each set of features made interoperable over the year. Most of the time, this leads to minor changes to the docs. MDN writers tried to describe the interoperable behaviour, such as only mentioning browsers when their implementations diverged.
```